src/pages/docs/courier/upgrade.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
how many blutui/cli user are their , if none do we need this information since everyone would be using courier

Reply:
You're right. I think its time we remove it. Perhaps we remove it from the menu but keep the page. Thoughts?

Reply:
Agreed